apps/Rename-It!/src/Gui/AboutDlg.cpp

```diff
@@ -1,6 +1,8 @@
 #include "StdAfx.h"
 #include "AboutDlg.h"
 
+//#define IDC_WEB_SO_ON_DEM 1133

```

Comment:
Should revert those comments (you can keep them on another branch if you don't want to lose them locally).